src/Silverback.Core/Messaging/Configuration/ServiceCollectionAddSilverbackExtensions.cs

```diff
@@ -1,74 +1,75 @@
-﻿// Copyright (c) 2020 Sergio Aquilini
```

Comment:
😐

Reply:
Why does the online editor show all lines as removed/added? What happened?